src/bin/test-all.ts

```diff
@@ -25,4 +32,10 @@ Promise.all(paths.map(path => glob(path)))
       debugLog(`(${Date.now() - startTime} ms) loaded ${match}`)
     })
   })
+
```

Comment:
Can we add a comment explaining all of this here:
 - index has to be loaded in the same syncronous chunk as test registry
 - If async before, it would load index and run 0 tests.
 - if async after, tests would run without the settings.

Reply:
bf490772307dc5c197350993e0779182aeaf9efd